Bulls_Cows.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+def generate_list():
+    lst = [i for i in range(10)]
+    lst = random.sample(lst, 4)
+    return lst
+
+def user_parse():
+        while True:
+            try:
+                user_try = list(input("Enter four number: "))
+                if len(user_try) != 4:
+                    raise Exception
```

Comment:
You should not use the `Exception` type. Rather use some of its subclasses. `ValueError` would be appropriate with some text
```
raise ValueError('You need to input a 4 digit number!')
```

But rather than raising an exception you can just `print` a text and `continue` because you are in an infinite loop.